wordpress-mobile/WordPress-Android
# Filter for images and video in add media picker

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1403

#### Companion PRs
gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1412

Can be merged after: https://github.com/wordpress-mobile/WordPress-Android/pull/10567 and https://github.com/wordpress-mobile/WordPress-Android/pull/10560

#### To test:
Follow steps listed here: https://github.com/WordPress/gutenberg/pull/17537#issue-320518695
and make sure videos and images can be selected from the picker after choosing "Choose from device".

Also, it should be possible to select videos and images after selecting "WordPress Media Library".

**Note:** When selecting via "WordPress Media Library", multiple items may be selected. This is a known issue, and the current behavior should be that a single image or video is used for the media-text block, while the rest are inserted below as image or video blocks. This issue is addressed in this PR: https://github.com/wordpress-mobile/WordPress-Android/pull/10597.

#### Screencasts

| Image | Video |
| :------------- | :------------- |
|<img src="https://user-images.githubusercontent.com/8507675/66288498-c7166300-e91c-11e9-8218-9e4bdde8ece6.gif" width="360"> | <img src="https://user-images.githubusercontent.com/8507675/66288503-cd0c4400-e91c-11e9-883b-99443d63bd5c.gif" width="360"> |


PR submission checklist:
